test/verify/check-realms

```diff
@@ -197,11 +197,25 @@ if ! grep -q 'services.*nss' /etc/sssd/sssd.conf; then
     systemctl restart sssd
 fi
 
-# This needs to work
-getent passwd admin@cockpit.lan
+# HACK: This needs to work, but may take a minute
+for x in $(seq 1 60); do
```

Comment:
Fortunately not 20! (that would be 10 billion years), just the sum (i. e. half an hour) :-)